build-presets/MakeBOSTree.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+import numpy as np
+from scipy.optimize import curve_fit
+import json
+
+efficiencies = []
+
+# 10% to 40% efficiency
+# building a list of efficiencies
+start = 0.1
+while (start < 0.41):
+	efficiencies.append(round(start, 2)) 
+	start += 0.01
+
+# creates a data frame for each system type
+bos_table = {
+	'fixed tilt, utility scale': pd.read_csv('BOS_cost_data/fixed_tilt_utility.csv'),
+	'single-axis tracked, utility scale': pd.read_csv('BOS_cost_data/tracked_utility.csv'),
+	'roof-mounted, residential scale': pd.read_csv('BOS_cost_data/roof_residential.csv'),
+	'roof-mounted, commercial scale': pd.read_csv('BOS_cost_data/roof_commercial.csv'),
+	'fixed tilt, commercial scale': pd.read_csv('BOS_cost_data/fixed_tilt_commercial.csv')
+}
+
+ilr = [1.1, 1.3, 1.4]
```

Comment:
it’s on the programmer to make sure the values are the same. but i don’t want to combine the trees since the BOS cost fit runs 15 times to make the BOS tree and the preset code runs pysam 3300 times to creates its presets. it seems silly to run the BOS cost fit 3285 unnecessary times, especially given how long the preset code takes to run already. I can leave a warning in the code that these values need to be the same between files